lib/game.js

```diff
@@ -128,5 +128,16 @@ Meteor.methods({
     check(Meteor.userId(), String);
     Meteor.users.update({_id: Meteor.userId()},
                         {$set: {'profile.name': name}});
-  }
+  },
+  'updateSetting': function (setting, value) {
+    check(Meteor.userId(), String);
+    // We want to set only this one profile setting (without overwriting the others). For some
+    // reason we get a compile-time error when trying to use a variable key in $set's dot notation
```

Comment:
javascript doesn't have syntax for object literals with non-constant keys, which is why you have to do this assignment explicitly.